mplc/scenario.py

```diff
@@ -61,21 +62,31 @@ def __init__(
         :param samples_split_option: ['basic', 'random'] (default),
                                      ['basic', 'stratified']
                                      or ['advanced', [[nb of clusters (int), 'shared' or 'specific']]].
-        :param corrupted_datasets: ['not_corrupted' (default), 'shuffled' or 'corrupted'].
-                                   Enables to artificially corrupt the data of one or several partners.
-                                   The size of the list must be equal to the number of partners
-
-                                   If one wants to shuffle only a part of the partner's data
-                                   it is possible to replace 'shuffled' by ['shuffled', proportion_shuffled]
-                                   where proportion_shuffled is a float between 0 and 1 indicating the
-                                   proportion of label to shuffle or corrupt within the partner.
-                                   Similarly for corruption it is possible to replace 'corrupted'
-                                   by ['corrupted', proportion_corrupted] where proportion_corrupted is a float
-                                   between 0 and 1 indicating the proportion of label to corrupt within the partner.
+        :param corruption_parameters: list of map.  Enables to artificially corrupt the data of one or several partners.
+                                   The size of the list must be equal to the number of partners.
+                                   Items of the list must be mapping. The possible keys, values are the following:
+                                    'corruption_method': 'not_corrupted' (default), 'duplication',
+                                                                                    'permutation',
+                                                                                    'permutation-circular',
+                                                                                    'random',
+                                                                                    'random-uniform' or 'redundancy'
+                                                                                    Indicating the corruption method
```

Comment:
To align properly all possible values

```suggestion
                                   Items of the list must be mapped. The possible keys, values are the following:
                                    'corruption_method': 'not_corrupted' (default),
                                                                         'duplication',
                                                                                    'permutation',
                                                                                    'permutation-circular',
                                                                                    'random',
                                                                                    'random-uniform',
                                                                                    'redundancy',
                                                                                    Indicating the corruption method
```